scripts/irods/test/test_rule_engine_plugin_metadata_guard.py

```diff
@@ -0,0 +1,328 @@
+from __future__ import print_function
```

Comment:
This can be removed.

Or we open an issue to remove python 2 code for a later release.

Reply:
Done